test/evergreen/evg_cfg.py

```diff
@@ -0,0 +1,335 @@
+#! /usr/bin/env python3
+
+"""
+This program provides an CLI interface to check and generate Evergreen configuration.
+"""
+
+import os
+import sys
+import re
+import subprocess
+import docopt
```

Comment:
My system didn't have `docopt` installed:
```
Traceback (most recent call last):
  File "/home/bostic/wiredtiger/test/evergreen/evg_cfg.py", line 11, in <module>
    import docopt
ModuleNotFoundError: No module named 'docopt'
```
We might want to add better error handling if this is a common/expected problem.

Reply:
Good suggestion. Will add some error handling here.